monitoring/zabbix_host.py

```diff
@@ -0,0 +1,458 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2013-2014, Epic Games, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+#
+
+DOCUMENTATION = '''
+---
+module: zabbix_host
+short_description: Zabbix host creates/updates/deletes
+description:
+   - When the host does not exists, a new host will be created, added to any host groups and linked to any templates.
+   - When the host already exists, the host group membership will be updated, along with the template links and interfaces.
+   - Delete a host from Zabbix if the host exists.
```

Comment:
i think it's being explicit that it's not just a dumb add/remove of the host, but it links and unlinks it properly.